### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

File: opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java [hunk 1]
```diff
@@ -140,13 +140,13 @@ public void onClick(DialogInterface dialog, int which) {
         }
     }
 
-    private void clearSavedSession() {
+    protected void clearSavedSession() {
```
Comment (ndegwamartin) [inline opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java:143]: Was this for testing only? If so we can add annotation _@VisibleForTesting_ here

File: opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java [hunk 2]
```diff
@@ -140,13 +140,13 @@ public void onClick(DialogInterface dialog, int which) {
...
         final OpdDiagnosisAndTreatmentForm opdDiagnosisAndTreatmentForm = new OpdDiagnosisAndTreatmentForm(OpdUtils.getIntentValue(getIntent(), OpdConstants.IntentKey.BASE_ENTITY_ID));
         final OpdDiagnosisAndTreatmentFormDao opdDiagnosisAndTreatmentFormDao = OpdLibrary.getInstance().getOpdDiagnosisAndTreatmentFormRepository();
         new AppExecutors().diskIO().execute(() -> opdDiagnosisAndTreatmentFormDao.delete(opdDiagnosisAndTreatmentForm));
     }
 
-    private void saveFormFillSession() {
+    protected void saveFormFillSession() {
```
Comment (ndegwamartin) [inline opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java:149]: Consider adding annotation _@VisibleForTesting_ here

**Review by ndegwamartin — APPROVED** [review]

File: opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java (hunk 2 above)
Comment (bennsimon) [reply opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java:152]: done

File: opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java (hunk 1 above)
Comment (bennsimon) [reply opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java:145]: done

**Review by ndegwamartin — APPROVED** [review]

**Pull request merged**